### ISSUE TITLE
The detail page shows an inactive remove state for locked addons only after page refresh

### ISSUE THREAD

**Issue by ioanarusiczki** [mozilla/addons-frontend#9365]
Followup for https://github.com/mozilla/addons-frontend/issues/8418

STR:
1. Using instructions for locking an addon go to https://github.com/mozilla/policy-templates/blob/master/README.md or https://github.com/mozilla/addons-frontend/issues/8418#issuecomment-603903349
2. After adding the addon id into the policies.json file open the browser and go to the detail page of the addon which is locked
3. Install the add-on and observe the page

Expected result:
"Remove" is inactive. - greyed out

Actual result:
"Remove" is greyed out only after page refresh or after restarting the browser.

Notes:
The "Enable" state is still active too and when it's clicked the "Remove" state will be displayed as being active too (as explained https://github.com/mozilla/addons-frontend/issues/8418#issuecomment-617164427)

"remove" after install

![page needs refresh2](https://user-images.githubusercontent.com/33448286/79871085-8976b480-83ec-11ea-9276-39ef2c570dab.gif)

"enable" when addon is locked 

![enable state not greyedout](https://user-images.githubusercontent.com/33448286/79870992-6c41e600-83ec-11ea-9981-59a185256916.gif)

**Comment by ioanarusiczki** [comment]
@willdurand 
I verified it on AMO dev and AMO stage - FF75(Win10) and the "remove" is automatically greyed out after installation.

For the second scenario - the "enable" state is still active. After it's clicked "remove" is inactive. Would be nice to have "enable" directly disabled on AMO. 

![can enable be greyed out](https://user-images.githubusercontent.com/33448286/79964715-c34ec600-8493-11ea-8f32-aff90af035b3.gif)

**Comment by willdurand** [comment]
> For the second scenario - the "enable" state is still active. After it's clicked "remove" is inactive. Would be nice to have "enable" directly disabled on AMO.

So this is actually the Firefox behavior. Sorry I thought I mentioned it somewhere, but if you apply your STR in FF, you'll see the add-on in the "disabled" section with the toggle switch available. It's possible to click on this toggle and it will enable the add-on.

**Comment by ioanarusiczki** [comment]
@willdurand After all an expected behavior because the addon should be enabled but not removed if it's locked.
I'm marking this verified on AMO dev and stage with FF75(Win10).
